packages/frontend/src/routes/SetupRecoveryImplicitAccountWrapper.js

```diff
@@ -60,7 +62,7 @@ export function SetupRecoveryImplicitAccountWrapper() {
                 Mixpanel.track('SR Verify email code');
                 try {
                     setVerifyingEmailCode(true);
-                    await wallet.validateSecurityCodeNewImplicitAccount(implicitAccountId, { kind: 'email', detail: email }, securityCode);
+                    await wallet.validateSecurityCodeNewImplicitAccount(implicitAccountId, { kind: 'email', detail: email }, securityCode, seedPhrasePublicKey);
```

Comment:
Since `handleInititalizeEmailRecoveryLink` does not get `await`'d and `showVerifyEmailCode` is set to `true` before it's called, I believe there's a race condition in which this can be called without a value for `publicKey`. Would it be safe to start `await`ing that method call and only setting `showVerifyEmailCode` once it's completed its API call?

Reply:
Yea I don't see any issues with awaiting the API call before setting `showVerifyEmailCode` to `true` - I suspect it'll be quick too. The only thing to consider is potentially adding a disabled/loading state for the main button in case the call takes some time to finish since right now we send the user to the 'verify' step right away.

Reply:
@Patrick1904 good call, I've added a flag to the component to disable the button during link initialization. Let me know if you think there's a more appropriate way to convey the loading.

Reply:
Sweet! nit: can we also please add `sending={isInitializingRecoveryLink}` as a prop to the `FormButton` so that we get that `Sending...` copy for the button?